overlord/state/change.go

```diff
@@ -477,6 +477,31 @@ func (c *Change) Tasks() []*Task {
 	return c.state.tasksIn(c.taskIDs)
 }
 
+// LaneTasks returns all tasks from given lanes the state change depends on.
+// All tasks are returned if any of the lanes is 0 (the default lane).
+func (c *Change) LaneTasks(lanes ...int) []*Task {
+	laneLookup := make(map[int]bool)
+	for _, l := range lanes {
+		if l == 0 {
```

Comment:
I said this, but then I think I said it's wrong...  we don't want this. The 0 lane is not a lane where all tasks of a change are, it's the lane of all the tasks that didn't join a specific set of lanes

Reply:
Ok, should be fixed now.